### PR TITLE
[js/web] FP16 binary and unary ops

### PR DESCRIPTION
**Pull request by dakenf**
### Description
Binary and unary ops with fp16 support

### CODE REVIEW

File: js/web/lib/wasm/jsep/webgpu/ops/unary-op.ts [hunk 1]
```diff
@@ -180,13 +184,13 @@ export const elu = (context: ComputeContext, attributes: AlphaAttributes): void
...
-const r5: f32 = 1.061405429;
+export const erfImpl = (dataType: string, varType = 'f32') => `
+const r0: ${varType} = 0.3275911;
+const r1: ${varType} = 0.254829592;
+const r2: ${varType} = -0.284496736;
+const r3: ${varType} = 1.421413741;
+const r4: ${varType} = -1.453152027;
+const r5: ${varType} = 1.061405429;
 
 fn erf_vf32(v: ${dataType}) -> ${dataType} {
```
Comment (fs-eire) [inline js/web/lib/wasm/jsep/webgpu/ops/unary-op.ts:195]: function name "erf_vf32" need to update

**Review by fs-eire — APPROVED** [review]

**Review by guschmue — APPROVED** [review]

**Pull request merged**